test/gateway/eip20_cogateway/confirm_stake_intent.js

```diff
@@ -91,8 +92,8 @@ contract('EIP20CoGateway.confirmStakeIntent() ', (accounts) => {
 
     assert.strictEqual(
       result,
-      data.messageHash,
-      `Message hash form the contract must match ${data.messageHash}.`,
+      messageHash,
+      `Message hash form the contract must match ${messageHash}.`,
```

Comment:
```suggestion
      `Message hash from the contract must match ${messageHash}.`,
```
Printing `${messageHash}` here is also redundant, as it will be part of the mocha/chai output. The message could simply be that "The contract did not return the expected message hash on `configmStakeIntent`."